patches/com/mojang/blaze3d/pipeline/RenderPipeline.java.patch

```diff
@@ -1,9 +1,64 @@
 --- a/com/mojang/blaze3d/pipeline/RenderPipeline.java
 +++ b/com/mojang/blaze3d/pipeline/RenderPipeline.java
-@@ -176,6 +_,46 @@
+@@ -41,7 +_,10 @@
+     private final float depthBiasConstant;
+     private final int sortKey;
+     private static int sortKeySeed;
++    private final Optional<net.neoforged.neoforge.client.stencil.StencilTest> stencilTest;
+ 
++    /** @deprecated Neo: use {@link #RenderPipeline(ResourceLocation, ResourceLocation, ResourceLocation, ShaderDefines, List, List, Optional, DepthTestFunction, PolygonMode, boolean, boolean, boolean, boolean, LogicOp, VertexFormat, VertexFormat.Mode, float, float, int, Optional)} instead */
++    @Deprecated
+     protected RenderPipeline(
+         ResourceLocation p_405403_,
+         ResourceLocation p_405389_,
+@@ -63,6 +_,31 @@
+         float p_409999_,
+         int p_415862_
+     ) {
++        this(p_405403_, p_405389_, p_404945_, p_405426_, p_405334_, p_404693_, p_404667_, p_410224_, p_409822_, p_404907_, p_405688_, p_405411_, p_405477_, p_410051_, p_405109_, p_404708_, p_410755_, p_409999_, p_415862_, Optional.empty());
++    }
++
++    protected RenderPipeline(
++            ResourceLocation p_405403_,
++            ResourceLocation p_405389_,
++            ResourceLocation p_404945_,
++            ShaderDefines p_405426_,
++            List<String> p_405334_,
++            List<RenderPipeline.UniformDescription> p_404693_,
++            Optional<BlendFunction> p_404667_,
++            DepthTestFunction p_410224_,
++            PolygonMode p_409822_,
++            boolean p_404907_,
++            boolean p_405688_,
++            boolean p_405411_,
++            boolean p_405477_,
++            LogicOp p_410051_,
++            VertexFormat p_405109_,
++            VertexFormat.Mode p_404708_,
++            float p_410755_,
++            float p_409999_,
++            int p_415862_,
++            Optional<net.neoforged.neoforge.client.stencil.StencilTest> stencilTest
```

Comment:
Nit: Would we rather keep the `sortKey` as the last parameter? No idea, probably doesn't matter anyway.

Reply:
Matty asked for it to be the last parameter (originally I did have sortKey left as the last param) for porting. Doesn't matter either way to me.